src/App.jsx

```diff
@@ -14,57 +16,62 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
```

Comment:
The name of your component 'App' is not very descriptive, it would be better to use a name that describes what the component does. This is related to the first point in the checklist: 'Method's name should start with a verb (so you could easily tell what your method actually do)'. In this case, it's not a method but a component, but the same principle applies. For example, a name like 'RenderGoods' or 'DisplayGoods' would be more descriptive.